src/CMake/unitTestSupport.cmake

```diff
@@ -9,7 +9,7 @@ execute_process(COMMAND "${CMAKE_COMMAND}" -E make_directory "${UNITTEST_RUN_BAS
 
```

Comment:
Copyright date (line 2) needs to be updated.  Being that we are AMD, you might wish to add the following line:
`Copyright (C) 2022 Advanced Micro Devices, Inc.`